build_mt.py

```diff
@@ -404,61 +391,33 @@ def main(sim_name, z_start, z_stop, merger_parent, catalog_parent, resume=False,
         # reorder file names by super slab number
         fns_this = reorder_by_slab(fns_this,minified)
         fns_prev = reorder_by_slab(fns_prev,minified)
+        
+        # maybe we want to support resuming from arbitrary superslab
+        first_ss = 0
+        
+        # We're going to be loading slabs in a rolling fashion:
+        # reading the "high" slab at the leading edge, discarding the trailing "low" slab
+        # and moving the mid to low. But first we need to read all three to prime the queue
+        mt_prev = {}  # indexed by slab num
+        mt_prev[(first_ss-1)%n_chunks] = get_mt_info(fns_prev[(first_ss-1)%n_chunks], fields=fields_mt, minified=minified)
+        mt_prev[first_ss] = get_mt_info(fns_prev[first_ss], fields=fields_mt, minified=minified)
```

Comment:
Here is where I start implementing the rolling version we talked about.  `mt_prev` contains any loaded mt_info's, keyed by slab number.